sections/problemFormulation.tex

```diff
@@ -0,0 +1,25 @@
+\newpage
+\section{Problem Formulation}
+
+Linear algebra calculations when sufficiently large can take a long time to compute sequentially. 
+Therefore it is smarter to parallelise these computations, and even better to do so on a GPU compared to a CPU, due to its internal architecture, described in \todo{GPU label}.
+Linear algebra calculations can be parallelise since none of the calculations done in for example multiplying 2 matrices, depend on previous calculations.
+Therefore these kinds of computations are an obvious choice for parallelisation on the GPU.
+
+Writing programs which can be run on a GPU, can be very time consuming using the languages found in \todo{SOTA kilde}.
+It is time consuming because of the difficult writing in some of these languages like OpenCL C, and CUDA.
```

Comment:
due to the difficulty of writing OR
due to the difficult languanges OpenCL C and CUDA